.github/workflows/fedora_29.yml

```diff
@@ -63,162 +63,38 @@ jobs:
             -DENABLE_SYSTEM_OPENCL=ON \
             -DENABLE_PYTHON_PACKAGING=ON \
             -DCPACK_GENERATOR=TGZ \
-            -DCMAKE_COMPILE_WARNING_AS_ERROR=ON \
-            -DCMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }} \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=${{ env.CMAKE_CXX_COMPILER_LAUNCHER }} \
-            -DCMAKE_C_COMPILER_LAUNCHER=${{ env.CMAKE_C_COMPILER_LAUNCHER }} \
-            -S ${OPENVINO_REPO} \
-            -B ${BUILD_DIR}
-
-      - name: Cmake build - OpenVINO
-        run: cmake --build ${BUILD_DIR} --parallel --verbose
-
-      - name: Show sccache stats
-        run: ${SCCACHE_PATH} --show-stats
-
-      - name: Cmake install - OpenVINO
-        run: |
-          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} -P ${BUILD_DIR}/cmake_install.cmake
-          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_TEST_DIR} -DCOMPONENT=tests -P ${BUILD_DIR}/cmake_install.cmake
-          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} -DCOMPONENT=python_wheels -P ${BUILD_DIR}/cmake_install.cmake
-
-      - name: Pack Artifacts
-        run: |
-          pushd ${INSTALL_DIR}
-            tar -czvf ${BUILD_DIR}/openvino_package.tar.gz *
-          popd
-
-          pushd ${INSTALL_TEST_DIR}
-            tar -czvf ${BUILD_DIR}/openvino_tests.tar.gz *
-          popd
-
-      - name: Build RPM packages
-        run: |
-          cmake -UTBB* \
-                -DCPACK_GENERATOR=RPM \
-                -DENABLE_SYSTEM_TBB=ON \
-                -DENABLE_TESTS=OFF \
-                ${BUILD_DIR}
-          cmake --build ${BUILD_DIR} --parallel --target package --verbose
-
-      #
-      # Upload build artifacts and logs
-      #
-      - name: Upload build logs
-        uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
-        if: always()
-        with:
-          name: build_logs
-          path: ${{ env.SCCACHE_ERROR_LOG }}
-          if-no-files-found: 'ignore'
-
-      - name: Upload openvino package
-        if: ${{ always() }}
-        uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
-        with:
-          name: openvino_package
-          path: ${{ env.BUILD_DIR }}/openvino_package.tar.gz
-          if-no-files-found: 'error'
-
-      - name: Upload openvino RPM packages
-        if: ${{ always() }}
-        uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
-        with:
-          name: openvino_rpm_packages
-          path: ${{ env.BUILD_DIR }}/*.rpm
-          if-no-files-found: 'error'
-
-      - name: Upload openvino tests package
-        if: ${{ always() }}
-        uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
-        with:
-          name: openvino_tests
-          path: ${{ env.BUILD_DIR }}/openvino_tests.tar.gz
-          if-no-files-found: 'error'
+            -DENABLE_INTEL_GPU=OFF \
+            -DENABLE_INTEL_NPU=OFF \
+            -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF \
```

Comment:
why some of the options changes their values?

Reply:
If they are not altered, there is a bunch of warnings/errors from npu, maybe INTEL_GPU is innocent:
![image](https://github.com/user-attachments/assets/07a4a7d1-b669-4f5e-8873-e03fab320bac)
 like in https://github.com/openvinotoolkit/openvino/actions/runs/10738251239/job/29781654015.